Makefile

```diff
@@ -10,7 +10,7 @@ RELEASE_BUILD ?= false
 CLOUD_PROVIDER ?=
 GOOPTIONS   ?= GOOS=linux GOARCH=$(ARCH) CGO_ENABLED=0
 GOFLAGS     ?=
-BINARIES    := cloud-api-adaptor agent-protocol-forwarder
+BINARIES    := cloud-api-adaptor agent-protocol-forwarder cluster-provisioner
```

Comment:
Is the cluster-provisioner binary ready for documenting yet to explain how to use it to set-up and peer pod enable the supported cloud providers?

Reply:
> > I guess some changes happened in CI recently? checking why errors happened like below in several job:
> > ```
> > cat: .git-commit: No such file or directory
> > ```
> 
> Yeah, maybe the `image` target needs a dependency on the `.git-commit` target now?

Thanks @stevenhorsman, I rerun `go mod tidy` actually but no more packages added in go.sum besides the items added already in the commits. Possible teh golang I'm using is dufferent as CI? will check it...

Reply:
> Is the cluster-provisioner binary ready for documenting yet to explain how to use it to set-up and peer pod enable the supported cloud providers?

Thanks @stevenhorsman I'll add some doc after testing it, the code has not been tested yet although binaries can be built.

Reply:
Great - I just wanted to understand what is, or isn't in scope for testing this PR :)